_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
+
+Είναι πράγματι πολύ δελεαστικό να χρησιμοποιήσει κανείς τα αυτοματοποιημένα εργαλεία ελέγχου για τη βαθμολόγηση του κώδικα των μαθητών. Ωστόσο, [[Nutb2016](#CITE)] αρχικά δεν βρήκε συσχετισμό μεταξύ των βαθμών που παρέχονται από ανθρώπινη βαθμολόγηση και των παραβιάσεων που προέρχονται από τα αυτόματα εργαλεία. Μερικές φορές αυτό οφείλεται στο γεγονός ότι οι μαθητές παραβίασαν ένα μόνο κανόνα πολλές φορές (χάνοντας έτσι περισσότερους βαθμούς από ό,τι θα έπρεπε) ενώ άλλες φορές αυτό οφείλεται επειδή υπέβαλαν τον κώδικα εκφώνησης με λίγες μόνο αλλαγές και επομένως πήραν μεγαλύτερο βαθμό από όσο θα έπρεπε.
+
+Οι [[Buff2015](#CITE)] παρουσιάζουν μια αρκετά ολοκληρωμένη σκέψη πάνω στην ιδέα της παροχής αυτοματοποιημένης ανατροφοδότησης. Το σημείο εκκίνησής τους είναι ότι "τα αυτοματοποιημένα συστήματα βαθμολόγησης βοηθούν τους εκπαιδευόμενους να εντοπίζουν τα σφάλματα μέσα τον κώδικα τους, [αλλά] μπορεί ταυτόχρονα να αποθαρρύνουν τους εκπαιδευόμενους από το να σκέφτονται κριτικά και να κάνουν ενδελεχείς ελέγχους, και αντ 'αυτού να ενθαρρύνουν την εξάρτηση από τα τεστ του δασκάλου". Ένα από τα καίρια ζητήματα που εντοπίστηκαν είναι ότι ο εκπαιδευόμενος μπορεί να πράγματι να ελέγξει διεξοδικά τον κώδικα του, αλλά η τελική υλοποίηση μπορεί να μην υπακούει στις προδιαγραφές του εκπαιδευτικού. Σε αυτή την περίπτωση, η "αποτυχία" δεν οφείλεται στην έλλειψη δοκιμών, αλλά στην παρανόηση των απαιτήσεων, και είναι αρκετά απίθανο η εφαρμογή επιπλεόν δοκιμών να εκθέσουν το πρόβλημα. Εάν το σύστημα αυτόματης αξιολόγησης δεν παρέχει διορατική, χρήσιμη και πρακτική ανατροφοδότηση, είναι πολύ πιθανό η συνολική εμπειρία να αποθαρρύνει τον εκπαιδευόμενο.
+
+Προκειμένου να δίνει τέτοιου είδους ανατροφοδότηση, το σύστημα των [[Buff2015](#CITE)] προσδιορίζει ποια μέθοδο ή μεθόδους του κώδικα του εκπαιδευόμενου οδηγούν στα αποτυχημένα τεστ, έτσι ώστε το σύστημα να μπορεί να συσχετίσει δοκιμές που απέτυχαν με συγκεκριμένα χαρακτηριστικά της υποβολής του εκπαιδευόμενου. Το σύστημα αποφασίζει εάν συγκεκριμένες νύξεις έχουν "κερδίσει" οι μαθητές, βλέποντας ένα έχουν πράγματι δοκιμάσει επαρκώς το σχετικό χαρακτηριστικό, ούτως ώστε οι εκπαιδευόμενοι να μην μπορούν να βασίζονται στις νύξεις αντί να κάνουν δοκιμές.
+
+Οι [[Keun2016a](#CITE),[Keun2016b](#CITE)] ταξινόμησαν τα μηνύματα που παράγουν 69 εργαλεία αυτόματης βαθμολόγησης. Διαπίστωσαν ότι αυτές συχνά δεν δίνουν ανατροφοδότηση στο πώς να διορθωθούν τα προβλήματα ώστε να γίνει το επόμενο βήμα. Βρήκαν επίσης ότι οι περισσότεροι εκπαιδευτικοί δεν μπορούν εύκολα να προσαρμόσουν περισσότερα από τα εργαλεία στις ανάγκες τους - όπως και πολλοί άλλα εργαλεία ροής εργασίας, τείνουν να επιβάλλουν τις μη ευσταθείς υποθέσεις των δημιουργών τους σχετικά με τον τρόπο λειτουργίας των εκπαιδευτικών οργανισμών. Οι συγγραφείς των άρθρων αυτών συνεχίζουν το έργο τους σε αυτό το χώρο και το λεπτομερές σύστημα ταξινόμησης τους είναι πολύ χρήσιμο για τον εντοπισμό των κατάλληλων εργαλείων αυτόματης βαθμολόγησης.
```

Comment:
ότι αυτές συχνά-->ότι αυτά συχνά
προσαρμόσουν περισσότερα-->προσαρμόσουν τα περισσότερα
πολλοί άλλα-->πολλά άλλα

Reply:
Τhanks @pdelias ! Fixed in a new commit.